airflow/config_templates/airflow_local_settings.py

```diff
@@ -82,7 +82,6 @@
             'class': 'airflow.utils.log.file_task_handler.FileTaskHandler',
             'formatter': 'airflow',
             'base_log_folder': os.path.expanduser(BASE_LOG_FOLDER),
-            'filename_template': FILENAME_TEMPLATE,
```

Comment:
We also need to leave this in case folks have an older provider, right?

Reply:
Probably, yes. Unless -- unless having extra parameters causes a problem for logging framework?

Reply:
Which we already do, so I think we are good here.

Reply:
Yeah. Custom logging config + backwards compatibiliyt is a dangerous combo